packages/ui/components/index.tsx

```diff
@@ -13,4 +13,7 @@ export * from './Sidebar'
 export * from './SlideOver'
 export * from './Toolbar'
 export * from './AppCard'
+export * from './Example'
+export * from './CodeEditor'
```

Comment:
Since this component is going to be the code editor of the visual editor, I would not export it directly here but as a subcomponent of VisualEditor. Just to avoid conflicting with the generic CodeEditor that we'll have in the future.

Reply:
sure. 👍